### DISCUSSION TITLE
[BaseAddress()]  Doesn't seem to work for me.

### DISCUSSION THREAD

**Discussion by psychicpuppy** [canton7/RestEase#268] [Q&A]
Added the [BaseAddress] attribute to my interface doesn't seem to work.  If I add the URL as an argument when creating the RestEase client it works.

**Comment by canton7** [comment]
If you want help finding out what your problem is, please provide more information.

**Reply by psychicpuppy** [reply]
I decorate my interface with the [BaseAddress] attribute and then create a RestEase client using the interface.  The resulting URL doesn't use the base address.  If I stick the base address URL as an argument when creating the RestEase client it works.

**Reply by canton7** [reply]
Please provide an actual code snippet which, when run, fully reproduces the problem.

I have plenty of tests which do what you describe, so I need to see exactly what you're doing, character for character.
